pkg/vsphere/session/session.go

```diff
@@ -77,6 +77,8 @@ type Config struct {
 	DatastorePath  string
 	HostPath       string
 	PoolPath       string
+
+	VCHFolder *object.Folder
```

Comment:
Even though the name is self-explanatory, a description for this exported field would be helpful - would be nice to note that this stores a different path depending on VC or ESX.

Reply:
Agree a comment is needed - this is the definitive VCH Appliance folder location...could be VMFolder could be a custom Folder...